components/homeRouter.js

```diff
@@ -1,19 +1,43 @@
-import { createSwitchNavigator, createStackNavigator, createAppContainer } from 'react-navigation';
-
+import React from 'react';
+import { createSwitchNavigator, createStackNavigator, createAppContainer, createBottomTabNavigator } from 'react-navigation';
+import { Icon } from 'react-native-elements';
 //Import screens
 import LogInOptionsScreen from "../screens/LogInOptions";
 import CreateAccount from "../screens/CreateAccount";
 import StartScreen from "../screens/Start";
 import HomeScreen from "../screens/Home"
 import SettingsScreen from "../screens/Settings";
 import WorkoutDetails from "../screens/WorkoutDetails"
+import Discover from "../screens/Discover";
+
+const TabNavigator = createBottomTabNavigator(
+  { 
+    Home: {
+      screen:HomeScreen,
+      navigationOptions:{
+        tabBarLabel:'Home',
+        tabBarIcon:() => (
+          <Icon name="home" size={22}/>
+        )
+      }
+    },
+    Discover: {
+      screen:Discover,
+      navigationOptions:{
+        tabBarLabel:'Discover',
+        tabBarIcon:() => (
+          <Icon name="md-compass" type='ionicon' size={22}/>
+        )
+      }
+    }
+  });
 
 //More about navigation https://reactnavigation.org/docs/en/auth-flow.html
 //createStackNavigator is a function that takes a route configuration object and an options object and returns a React component.
 const AppStack = createStackNavigator(
   { 
     UserLibrary: {
-      screen: HomeScreen,
+      screen: TabNavigator,
```

Comment:
User Library is a Home screen, not a tab navigation.

Reply:
@AlexeiGitH  Yes. However, when I tried to include createBottonTabNavigator in createAppContainer it did not work. I think CreateAppContainer can only handle one event and since it already has createSwitchNavigator, including createBottonTabNavigator did not work. Hence, one way to handle both events is to have UserLibrary screen as TabNavigator.

Reply:
Did you look at the example? 
Don't want to spend time on this, I am going to merge and upload the fix for it.